### PR TITLE
ENH Add localisation support

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Description
Added localisation support. All available messages, field names, etc. are updated and have the value set in the localisation file

## Parent issue
- https://github.com/silverstripe/silverstripe-linkfield/issues/113

### CODE REVIEW

File: lang/en.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+en:
+  LinkField:
+    UPDATELINK: 'Update link'
```
Comment (emteknetnz) [inline lang/en.yml]: There isn't a clear standard around translation keys. While uppercase without underscores is the most common, is also hard to read

Could you please put underscores in the keys e.g. `UPDATE_LINK`

File: src/Models/ExternalLink.php [hunk 2]
```diff
@@ -15,6 +17,15 @@ class ExternalLink extends Link
         'ExternalUrl' => 'Varchar',
     ];
 
+    public function getCMSFields(): FieldList
+    {
+        $this->beforeUpdateCMSFields(static function (FieldList $fields) {
```
Comment (emteknetnz) [inline src/Models/ExternalLink.php]: Could you please change `static function` to just `function` for all of the anonymous methods in the getCMSFields for all of the subclasses on Link

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: lang/en.yml (hunk 1 above)
Comment (sabina-talipova) [reply lang/en.yml]: Updated

File: src/Models/ExternalLink.php (hunk 2 above)
Comment (sabina-talipova) [reply src/Models/ExternalLink.php]: Updated

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**